tests/contrib/redis/test_redis_cluster.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import pytest
+import redis
+
+from ddtrace import Pin
+from ddtrace.contrib.redis.patch import patch
+from ddtrace.contrib.redis.patch import unpatch
+from tests.contrib.config import REDISCLUSTER_CONFIG
+from tests.utils import DummyTracer
+from tests.utils import TracerTestCase
+from tests.utils import assert_is_measured
+
+
+@pytest.mark.skipif(redis.VERSION < (4, 1), reason="redis.cluster is not implemented in redis<4.1")
+class TestRedisClusterPatch(TracerTestCase):
```

Comment:
this test is based on `TestGrokzenRedisClusterPatch`, but i've decided to copy-paste, instead of inheriting due to the reasons below.
- `rediscluster` is deprecated, so there is no need to change `TestGrokzenRedisClusterPatch`. However, `redis.cluster` will be updated in the future, so there will be a need to fix tests from time to time
- different package import is required, so there was to be a change in almost every function, so rather than putting lots of `if else`, copy-pasting looked better
  - span `service` changed from `rediscluster` to `redis`
  - patch, unpatch and RedisCluster requires import from different package
  - env `DD_REDISCLUSTER_SERVICE` changed to `DD_REDIS_SERVICE`